pennylane/devices/execution_config.py

```diff
@@ -20,6 +20,22 @@
 from pennylane.workflow import SUPPORTED_INTERFACES
 
 
+@dataclass
+class MCMConfig:
+    """A class to store mid-circuit measurement configurations."""
+
+    mcm_method: Optional[str] = None
+    """Which mid-circuit measurement strategy to use. Use ``deferred`` for the deferred
+    measurements principle and "one-shot" if using finite shots to execute the circuit
+    for each shot separately."""
+
+    postselect_mode: Optional[str] = None
```

Comment:
Same question here. What does `None` indicate?

Reply:
Actually, the default now should be `"hw-like"`. I'll update that wherever necessary.